addons/account/migrations/9.0.1.1/post-migration.py

```diff
@@ -0,0 +1,362 @@
+            pieces = x['pieces']
+            cr.execute("""
+            INSERT INTO account_cashbox_line (cashbox_id, number, coin_value) VALUES (%(cash_id)s, %(opening_number)s, %(pieces)s) 
+            """ %{'opening_number' : opening_number, 'pieces' : pieces, 'cash_id' : cashbox_id})
+
+        cr.execute("""
+        UPDATE account_bank_statement SET cashbox_start_id = %s WHERE id = %s
+        """ %(cashbox_id, bank_statement_id))
+
+        cr.execute("""
+        SELECT pieces, number_closing FROM account_cashbox_line WHERE number_closing IS NOT NULL AND bank_statement_id  = %s
+        """ %bank_statement_id)
+
+        closing_cashbox = cr.dictfetchall()
+
+        cr.execute("""
+        INSERT INTO account_bank_statement_cashbox (create_date) VALUES (NULL) RETURNING id
+        """)
+
+        cashbox_id = cr.fetchone()[0]
+
+        for x in closing_cashbox:
+            closing_number = x['number_closing']
+            pieces = x['pieces']
+            cr.execute("""
+            INSERT INTO account_cashbox_line (cashbox_id, number, coin_value) VALUES (%(cash_id)s, %(closing_number)s, %(pieces)s) 
+            """ %{'closing_number' : closing_number, 'pieces' : pieces, 'cash_id' : cashbox_id})
+
+        cr.execute("""
+        UPDATE account_bank_statement SET cashbox_end_id = %s WHERE id = %s
+        """ %(cashbox_id, bank_statement_id))
+        
+def account_properties(cr):
+    # Handle account properties as their names are changed.
+    cr.execute("""
+            update ir_property set name = 'property_account_payable_id', 
+            fields_id = (select id from ir_model_fields where model 
+            = 'res.partner' and name = 'property_account_payable_id') 
+            where name = 'property_account_payable' and (res_id like 
+            'res.partner%' or res_id is null)
+            """)
+    cr.execute("""
+            update ir_property set fields_id = (select id from 
+            ir_model_fields where model = 'res.partner' and 
+            name = 'property_account_receivable_id'), name = 
+            'property_account_receivable_id' where 
+            name = 'property_account_receivable' and (res_id like 
+            'res.partner%' or res_id is null)
+            """)
+
+@openupgrade.migrate()
+def migrate(cr, version):
+    map_bank_state(cr)
+    map_type_tax_use(cr)
+    map_type_tax_use_template(cr)
+    map_journal_state(cr)
+    account_templates(cr)
+    parent_id_to_m2m(cr)
+    cashbox(cr)
+    account_properties(cr)
+
+    # If the close_method is 'none', then set to 'False', otherwise set to 'True'
+    cr.execute("""
+    UPDATE account_account_type SET include_initial_balance =  CASE
+    WHEN %(openupgrade)s = 'none' THEN False
+    ELSE True
+    END 
+    """%{'openupgrade' : openupgrade.get_legacy_name('close_method')})
+
+    # Set bank_statements_source to 'manual' 
+    cr.execute("""
+    UPDATE account_journal SET bank_statements_source = 'manual'
```

Comment:
for things like this, you can use https://doc.therp.nl/openupgrade/API.html#openupgradelib.openupgrade.set_defaults